src/openvic-simulation/economy/ProductionType.cpp

```diff
@@ -175,46 +181,46 @@ bool ProductionTypeManager::load_production_types_file(
 				return true;
 			}
 
-			EmployedPop owner;
-			std::vector<EmployedPop> employees;
-			ProductionType::type_t type;
+			Job owner {};
+			std::vector<Job> jobs;
+			ProductionType::template_type_t template_type;
```

Comment:
```suggestion
			ProductionType::template_type_t template_type = FACTORY;
```
Again, safer to have a default value, and you can move `using enum ProductionType::template_type_t;` to the top of the function to avoid needing the enum's prefix.